pkg/agent/multicast/mcast_discovery.go

```diff
@@ -38,7 +38,7 @@ const (
 	// queryInterval is the interval to send IGMP query messages.
 	queryInterval = time.Second * 125
```

Comment:
If a configurable item is added, you should remove this constant, and use the configuration in the caller? So does `mcastGroupTimeout`

Reply:
I remain this value for unit test, do you mean use a variable directly in unit test than the const?

Reply:
Yes.